MediumProject/src/main/java/stage1/Searching.java

```diff
@@ -0,0 +1,27 @@
+package stage1;
+
+import java.util.Scanner;
+
+public class Searching {
+    private static String[] inputLine(){
+        Scanner scan = new Scanner(System.in);
+        String inputLine = scan.nextLine();
+        return inputLine.split(" ");
+    }
+    private static String inputWord(){
+        Scanner scan = new Scanner(System.in);
+        return scan.nextLine();
+    }
+
+    public static void search(){
+        String[] arr = inputLine();
+        String word = inputWord();
+        String result = " ";
+        for (int i = 0; i < arr.length; i++) {
+            if (arr[i].equals(word)) result = Integer.toString(i+1);
```

Comment:
why? Integer.toString(i+1)

Reply:
okay you should return index

Reply:
just for alternative

        var var1 =IntStream.range(0, arr.length)
                .filter(index -> arr[index].equals(word))
                .mapToObj(index -> Integer.valueOf(index).toString())
                .findFirst()
                .orElse("Not found");
        System.out.println(var1);